schema/claim-result.schema.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$id": "https://cnab.io/v1/claim-result.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "claimRef": {
```

Comment:
Should this `claimRef` have any required fields (`installation` or `revision` or both?) or acceptable to be empty?  (`claimRef` itself is designated as required on this claim-result)